src/Data/Document.hs

```diff
@@ -0,0 +1,89 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Data.Document (
+  Document (Document),
```

Comment:
I would probably just write `Document (..)` to export the constructors and record fields. I'm curious why you would write it this way.

Reply:
Because I didn't know  `Document (..)` was a possibility and when I asked _How do I export a data constructor?_, this is the answer I got.

Thanks for setting me right :)

Reply:
Should be fixed in 24e7bde2bcc268d64f3c32bbab00daa4b263af27.